.github/workflows/node.js.yml

```diff
@@ -21,6 +21,8 @@ jobs:
         env:
             VSCODE_TEST_VERSION: ${{ matrix.vscode-version }}
             NODE_OPTIONS: '--max-old-space-size=8192'
+            # Set a fixed timezone for tests. Intentionally _not_ UTC.
+            TZ: 'US/Pacific'
```

Comment:
No other way (AFAICT) to force a timezone in nodejs/javascript.

TODO: Ideally we could do this in e.g. `launchTestUtilities.ts` but I think it's required at startup, can't change timezone of the running process.

Reply:
I was playing around with setting `process.env.tz = "US/Pacific"` in `mochaGlobalSetup()`. Does that work for you?

Reply:
seems to work!

Reply:
Looks like it could be due to the global setup function running too late (`script/test/test.ts` runs before it in CI) and there is a race between defining the env var and when node "locks" it in on the first read
`"you can set the TZ environment variable before calling any date functions"`
https://stackoverflow.com/questions/8083410/how-can-i-set-the-default-timezone-in-node-js

I wonder if setting `process.env.TZ` in the global scope in `test/launchUtilities.ts` (or scripts/test/tests.ts) will get it working since they are the files first executed

Reply:
> I wonder if setting `process.env.TZ` in the global scope in `test/launchUtilities.ts` (or scripts/test/tests.ts) will get it working since they are the files first executed

I will try that in a followup.